src/Controller/TalkCommentsController.php

```diff
@@ -15,6 +15,11 @@
 class TalkCommentsController extends BaseApiController
 // @codingStandardsIgnoreEnd
 {
+    /**
+     * @var TalkCommentMapper
+     */
+    protected $commentMapper;
```

Comment:
Will `private` visibility suffice here?

Reply:
Should do, not sure why it's `protected`. I never make anything `protected`, lol.